tests/foreman/ui/test_contentview.py

```diff
@@ -102,7 +102,7 @@
     YumRepository,
 )
 from robottelo.vm import VirtualMachine, VirtualMachineError
-
+from productmd.common import parse_nvra
```

Comment:
Nitpick: according to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports), import order should be: standard library, third party, local library.
To be aligned with PEP8 recommendation, this import should be moved to line 29.

Reply:
I've resolved this.